code/game/gamemodes/cult/items/tome.dm

```diff
@@ -0,0 +1,200 @@
+	visible_message(span("warning", "\The [user] beats \the [M] with \the [src]!"))
+	to_chat(M, span("danger", "You feel searing heat inside!"))
+	attack_admins(M, user)
+
+/obj/item/book/tome/proc/attack_admins(var/mob/living/M, var/mob/living/user)
+	M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Has had the [name] used on them by [user.name] ([user.ckey])</font>")
+	user.attack_log += text("\[[time_stamp()]\] <font color='red'>Used [name] on [M.name] ([M.ckey])</font>")
+	msg_admin_attack("[key_name_admin(user)] used [name] on [M.name] ([M.ckey]) (<A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[user.x];Y=[user.y];Z=[user.z]'>JMP</a>)",ckey=key_name(user),ckey_target=key_name(M))
+
+
+/obj/item/book/tome/attack_self(mob/living/user)
+	if(use_check_and_message(user))
+		return
+
+	if(iscultist(user))
+		if(!istype(user.loc, /turf))
+			to_chat(user, span("warning", "You do not have enough space to write a proper rune."))
+			return
+
+		var/turf/T = get_turf(user)
+
+		if(T.is_hole || T.is_space())
+			to_chat(user, span("warning", "You are unable to write a rune here."))
+			return
+
+		// This counts how many runes exist in the game, for some sort of arbitrary rune limit. I trust the old devs had their reasons. - Geeves
+		if(length(rune_list) >= 25 + rune_boost + cult.current_antagonists.len)
+			to_chat(user, span("warning", "The cloth of reality can't take that much of a strain. Remove some runes first!"))
+			return
+		else
+			switch(alert("What shall you do with the tome?", "Tome of Nar'sie", "Read it", "Scribe a rune", "Cancel"))
+				if("Cancel")
+					return
+				if("Read it")
+					if(use_check_and_message(user))
+						return
+					user << browse("[tomedat]", "window=Arcane Tome")
+					return
+
+		//only check if they want to scribe a rune, so they can still read if standing on a rune
+		if(locate(/obj/effect/rune) in user.loc)
+			to_chat(user, span("warning", "There is already a rune in this location."))
+			return
+
+		if(use_check_and_message(user))
+			return
+
+		var/chosen_rune
+		var/network
+		chosen_rune = input("Choose a rune to scribe.") in rune_types
+		if(!chosen_rune)
+			return
+		if(chosen_rune == "None")
+			to_chat(user, span("notice", "You decide against scribing a rune, perhaps you should take this time to study your notes."))
+			return
+		if(chosen_rune == "Teleport")
+			network = input(user, "Choose a teleportation network for the rune to connect to.", "Teleportation Rune") in teleport_network
+
+		if(use_check_and_message(user))
+			return
+
+		user.visible_message(span("warning", "\The [user] slices open a finger and begins to chant and paint symbols on the floor."), span("notice", "You slice open one of your fingers and begin drawing a rune on the floor whilst softly chanting the ritual that binds your life essence with the dark arcane energies flowing through the surrounding world."))
+		user.take_overall_damage((rand(9)+1)/10) // 0.1 to 1.0 damage
```

Comment:
At this point, it might be worth making this into adding a bleeding wound instead.

Reply:
Too much shitcode to implement that currently.